nuxeo-liveconnect-box-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-liveconnect/nuxeo-liveconnect-box-link.html

```diff
@@ -45,6 +45,17 @@
         @apply --layout-center;
         @apply --layout-center-justified;
         @apply --nuxeo-liveconnect-anchor-layout;
+        color: var(--nuxeo-secondary-color, #0066ff);
+      }
+
+      a:hover {
+        color: var(--nuxeo-link-hover_-_color);
+        cursor: var(--nuxeo-link-hover_-_cursor);
+      }
+
+      a, a:active, a:visited, a:focus {
```

Comment:
Since we want to keep coherency with [nuxeo-dropzone](https://github.com/nuxeo/nuxeo-web-ui/blob/master/elements/nuxeo-dropzone/nuxeo-dropzone.html), it may be useful to take a look to own it is being done there.